File: PYTHON/Program/Clothing.py
# memanggil class product
from Product import Product

# membuat child dari class product
class Clothing(Product):
    __size = ""
    __material = ""
    __gender = ""

    def __init__(self, idProduct="", name="", brand ="", price="", size="", material="", gender=""):
        super().__init__(idProduct, name, brand, price)
        self.__size = size
        self.__material = material
        self.__gender = gender

    # Fungsi get
    def get_size(self):
        return self.__size

    def get_material(self):
        return self.__material

    def get_gender(self):
        return self.__gender

    # Fungsi set
    def set_size(self, size):
        self.__size = size

    def set_material(self, material):
        self.__material = material

    def set_gender(self, gender):
        self.__gender = gender

File: CPP/Program/Clothing.cpp
// +==========================================================+
// +                                                          +
// + Saya Raden Rahman Ismail (2200311)                       +
// + mengerjakan soal Latihan dalam mata kuliah DPBO          +
// + untuk keberkahanNya maka saya tidak melakukan kecurangan +
// + seperti yang telah dispesikasikan.Aamiin                 +
// +                                                          +
// +==========================================================+

// memanggil libary yang digunakan dan memanggil class product
#include <iostream>
#include <string>
#include "Product.cpp"

// using standard namespace
using namespace std;

// membuat class Clothing
class Clothing : public Product
{
private:
    string size;
    string material;
    string gender;

public:
    Clothing()
    {
        this->size = "";
        this->material = "";
        this->gender = "";
    };

    Clothing(string size, string material, string gender)
    {
        this->size = size;
        this->material = material;
        this->gender = gender;
    }

    // Fungsi get
    string getSize()
    {
        return this->size;
    }
    string getMaterial()
    {
        return this->material;
    }
    string getGender()
    {
        return this->gender;
    }

    // fungsi set
    void setSize(string size)
    {
        this->size = size;
    }
    void setMaterial(string material)
    {
        this->material = material;
    }
    void setGender(string gender)
    {
        this->gender = gender;
    }

    ~Clothing()
    {
    }
};
